WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -27,7 +27,8 @@ public class CoinJoinClient
 {
 	private const int MaxInputsRegistrableByWallet = 10; // how many
 	private static readonly Money MinimumOutputAmountSanity = Money.Coins(0.0001m); // ignore rounds with too big minimum denominations
-	private static readonly TimeSpan ExtraPhaseTimeoutMargin = TimeSpan.FromMinutes(1);
+	private static readonly TimeSpan ExtraPhaseTimeoutMargin = TimeSpan.FromMinutes(2);
```

Comment:
This variable is currently not used.

Reply:
I don't see what you are saying.

Reply:
![image](https://user-images.githubusercontent.com/9844978/188511518-35fc44a8-9025-48d9-a84d-c139c3514174.png)